app/server/postgraphile/postgraphileOptions.js

```diff
@@ -1,14 +1,15 @@
 const {makePluginHook} = require('postgraphile');
 const PgManyToManyPlugin = require('@graphile-contrib/pg-many-to-many');
 const PostgraphileLogConsola = require('postgraphile-log-consola');
+const ConnectionFilterPlugin = require('postgraphile-plugin-connection-filter');
 
 const postgraphileOptions = () => {
   // Use consola for logging instead of default logger
   const pluginHook = makePluginHook([PostgraphileLogConsola]);
 
   let postgraphileOptions = {
     pluginHook,
-    appendPlugins: [PgManyToManyPlugin],
+    appendPlugins: [PgManyToManyPlugin, ConnectionFilterPlugin],
```

Comment:
can those go in the `.postgraphilerc.js` file? This would avoid having to maintain the same list in the `package.json`